diff -aurN gdm-3.6.1.orig/gui/simple-chooser/gdm-chooser-session.c gdm-3.6.1/gui/simple-chooser/gdm-chooser-session.c
--- gdm-3.6.1.orig/gui/simple-chooser/gdm-chooser-session.c	2012-11-01 13:52:27.367552257 +0100
+++ gdm-3.6.1/gui/simple-chooser/gdm-chooser-session.c	2012-11-01 13:55:29.416977906 +0100
@@ -133,7 +133,7 @@
         ret = FALSE;
 
         error = NULL;
-        g_spawn_command_line_async (LIBEXECDIR "/gnome-settings-daemon", &error);
+        g_spawn_command_line_async (LIBDIR "gnome-settings-daemon-3.0/gnome-settings-daemon", &error);
         if (error != NULL) {
                 g_warning ("Error starting settings daemon: %s", error->message);
                 g_error_free (error);
diff -aurN gdm-3.6.1.orig/gui/simple-chooser/Makefile.am gdm-3.6.1/gui/simple-chooser/Makefile.am
--- gdm-3.6.1.orig/gui/simple-chooser/Makefile.am	2012-11-01 13:52:27.367552257 +0100
+++ gdm-3.6.1/gui/simple-chooser/Makefile.am	2012-11-01 14:01:36.435359644 +0100
@@ -10,6 +10,7 @@
 	-DDMCONFDIR=\""$(dmconfdir)"\"		 	\
 	-DGDMLOCALEDIR=\""$(gdmlocaledir)"\"		\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
+	-DLIBDIR=\""$(libdir)"\" 			\
 	-DLIBEXECDIR=\""$(libexecdir)"\" 		\
 	-DSBINDIR=\""$(sbindir)"\"		 	\
 	-DPIXMAPDIR=\""$(pixmapdir)"\"		 	\
