diff -aurN gdm-3.8.0.orig/data/gdm-shell-with-fallback.session.in gdm-3.8.0/data/gdm-shell-with-fallback.session.in
--- gdm-3.8.0.orig/data/gdm-shell-with-fallback.session.in	2013-03-29 17:28:33.062008055 +0100
+++ gdm-3.8.0/data/gdm-shell-with-fallback.session.in	2013-03-29 17:29:26.683001316 +0100
@@ -1,5 +1,5 @@
 [GNOME Session]
 Name=Display Manager
 RequiredComponents=gnome-shell;gnome-settings-daemon;
-IsRunnableHelper=bash -c 'type -p gnome-shell >& /dev/null && @CHECK_ACCELERATED_DIR@/gnome-session-check-accelerated'
+IsRunnableHelper=sh -c 'type -p gnome-shell >& /dev/null && @CHECK_ACCELERATED_DIR@/gnome-session-check-accelerated'
 FallbackSession=gdm-fallback
